src/client/Microsoft.Identity.Client/Internal/SsoPolicy/ISsoPolicy.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+namespace Microsoft.Identity.Client.Internal.SsoPolicy
+{
+    internal interface ISsoPolicy
```

Comment:
empty interfaces are strange :)

Reply:
this was draft and anways the new approach is different reusing the same IBroker interface.